src/rpc/net.cpp

```diff
@@ -165,12 +165,12 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
         obj.pushKV("bytesrecv", stats.nRecvBytes);
         obj.pushKV("conntime", stats.nTimeConnected);
         obj.pushKV("timeoffset", stats.nTimeOffset);
-        if (stats.dPingTime > 0.0)
-            obj.pushKV("pingtime", stats.dPingTime);
-        if (stats.dMinPing < static_cast<double>(std::numeric_limits<int64_t>::max())/1e6)
-            obj.pushKV("minping", stats.dMinPing);
-        if (stats.dPingWait > 0.0)
-            obj.pushKV("pingwait", stats.dPingWait);
+        if (stats.m_ping_usec > 0)
+            obj.pushKV("pingtime", stats.m_ping_usec / 1e6);
```

Comment:
Should cast to double first?

Reply:
Done

Reply:
Assuming `x` is `int64_t`, then all of these are equivalent:

```
x / 1e6;
(double)x / 1e6;
((double)x) / 1e6;
x / 1000000.0;
```

Reply:
Indeed, I tend to prefer the explicit numerator cast though.